### PR TITLE
Animation Cleanup: En_K*

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/overlays/actors/ovl_En_Kbt/z_en_kbt.c [hunk 1]
```diff
@@ -82,27 +99,48 @@ s32 func_80B33E8C(PlayState* play) {
...
+        &object_kbt_Anim_002DE0, // ENKBT_ANIM_8
+        &object_kbt_Anim_003414, // ENKBT_ANIM_9
+        &object_kbt_Anim_003D24, // ENKBT_ANIM_10
+        &object_kbt_Anim_001BF4, // ENKBT_ANIM_11
+        &object_kbt_Anim_002084, // ENKBT_ANIM_12
     };
-    u8 sp38[] = {
-        ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_ONCE, ANIMMODE_LOOP, ANIMMODE_ONCE, ANIMMODE_LOOP, ANIMMODE_LOOP,
-        ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_ONCE, ANIMMODE_ONCE,
+    u8 animationModes[ENKBT_ANIM_MAX] = {
```
Comment (tom-overton) [inline src/overlays/actors/ovl_En_Kbt/z_en_kbt.c:118]: This one *isn't* `static`, but the previous one is. Is that required to match? Or can we make them consistent?

**Review by tom-overton — COMMENTED** [review]
Just one small thing.

File: src/overlays/actors/ovl_En_Kbt/z_en_kbt.c (hunk 1 above)
Comment (engineer124) [reply src/overlays/actors/ovl_En_Kbt/z_en_kbt.c:118]: No, it has to be this way unfortunately. `sAnimations` needs to be in-function and static, and `animationModes` needs to be in-function and non-static

Comment (tom-overton) [reply src/overlays/actors/ovl_En_Kbt/z_en_kbt.c:118]: Ah, oh well. Thanks for checking!

**Review by tom-overton — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Review by isghj5 — APPROVED** [review]

**Pull request merged**